### PR TITLE
Implement export to org

### LINKED ISSUE #8: Implement export command
Export to org for now.

### PR DESCRIPTION
**Pull request by filipelbc**
Closes #8 

Also made some fixes.

### CODE REVIEW

File: dit.py [hunk 1]
```diff
@@ -279,60 +281,60 @@ def _rebuild_index(self):
     # ===========================================
     # Export
 
-    def _export_task_(self, printer, group, subgroup, task, task_id, concluded, verbose):
+    def _export_task_(self, group, subgroup, task, task_id, concluded, verbose):
```
Comment (danielmoraes) [inline dit.py]: Missing arguments "group_id" and "subgroup_id".

File: dit.py [hunk 2]
```diff
@@ -279,60 +281,60 @@ def _rebuild_index(self):
...
         data = self._get_task_data(group, subgroup, task)
 
         if not data.get('concluded_at', None) or concluded:
-            printer.task(group, subgroup, task, task_id, data, verbose)
+            self.printer.task(group, subgroup, task, task_id, data, verbose)
```
Comment (danielmoraes) [inline dit.py]: Missing parameters "group_id" and "subgroup_id".

**Comment by danielmoraes** [comment]
Not listing/exporting group/subgroup "0".

File: dit.py [hunk 3]
```diff
@@ -279,60 +281,60 @@ def _rebuild_index(self):
...
 
-    def _export_task(self, printer, group, subgroup, task, concluded, verbose):
+    def _export_task(self, group, subgroup, task, concluded, verbose):
         for i in range(len(self.index)):
             if self.index[i][0] == group:
                 for j in range(len(self.index[i][1])):
                     if self.index[i][1][j][0] == subgroup:
                         for k in range(self.index[i][1][j][1]):
```
Comment (danielmoraes) [inline dit.py]: Missing len() function.

Comment (danielmoraes) [inline dit.py]: Missing len() function inside range().

**Comment by danielmoraes** [comment]
Empty groups/subgroups should not be listed/exported.

**Comment by filipelbc** [comment]
This decision should not be taken in this loop, but in the printer. And this should be an option. So, create an issue for that.

**Pull request merged**